src/server/models/list.ts

```diff
@@ -200,6 +215,49 @@ export async function updateList(
   }
 }
 
+export async function removeUserFromList(listId: number, userEmail: string): Promise<void> {
+  try {
+    await prisma.list.update({
+      where: {
+        id: listId,
+      },
+      data: {
+        users: {
+          disconnect: {
+            email: userEmail,
+          },
+        },
+      },
+    });
+    logger.info(`User ${userEmail} disconnected from list ${listId}`);
+  } catch (error) {
+    logger.error(`removeUserFromList Error: ${(error as Error).message}`);
+    throw error;
+  }
+}
+
+async function createUsersFromEmails(email: string): Promise<void> {
+  await prisma.user.create({
+    data: {
+      email,
+      jsonData: {
+        roles: [],
+      },
+    },
+  });
+
+  logger.info(`Created users for ${email}`);
+}
+
+async function checkUserExists(email: string): Promise<User> {
+  const res = await prisma.user.findUnique({
```

Comment:
Ideally just let TS infer prisma queries 

```suggestion
async function checkUserExists(email: string) {
  return await prisma.user.findUnique({
```

☝️ prisma generates a complex type system. If we allow typescript to infer instead of casting, the inferred type will be more "true" to the query input. e.g. If the query had "select: { id: true }" - then the inferred type would only include id.